src/main/java/mate/academy/Main.java

```diff
@@ -2,15 +2,13 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
-        // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(5);
```

Comment:
Consider specifying the number of threads for the ExecutorService based on the number of available processors, as this could lead to better performance. For example, you could use `Runtime.getRuntime().availableProcessors()` to get the number of processors.